src/Model/Actions.re

```diff
@@ -27,7 +27,7 @@ type t =
   | BufferEnter([@opaque] Vim.BufferMetadata.t, option(string))
   | BufferUpdate([@opaque] BufferUpdate.t)
   | BufferRenderer(BufferRenderer.action)
-  | BufferSaved([@opaque] Vim.BufferMetadata.t)
+  | BufferSaved([@opaque] int)
```

Comment:
Why make the `int` opaque?

Reply:
Ah, good point, I left that in after switching to the `int`. Removed now